packages/vuetify/src/mixins/menuable/index.ts

```diff
@@ -14,14 +14,83 @@ const baseMixins = mixins(
   Activatable
 )
 
+interface dimensions {
+  top: number
+  left: number
+  bottom: number
+  right: number
+  width: number
+  height: number
+  offsetTop: number
+  scrollHeight: number
+  offsetLeft: number
+}
+
 interface options extends ExtractVue<typeof baseMixins> {
+  hasActivator: boolean
   attach: boolean | string | Element
   offsetY: boolean
   offsetX: boolean
+  computedRelativeOffset: { top: number, left: number }
```

Comment:
This is unnecessary

Reply:
You mean the type declaration or just this method? Without them typescript is not happy at all